client/my-sites/plugins/plugins-discovery-page/upgrade-nudge.jsx

```diff
@@ -132,17 +132,18 @@ const UpgradeNudge = ( {
 	}
 
 	const title = translate(
-		'You need to upgrade to a %(businessPlanName)s Plan to install plugins. Get a free domain with an annual plan.',
+		'You need to upgrade to a %(businessPlanName)s Plan to install plugins.',
 		{ args: { businessPlanName: getPlan( plan )?.getTitle() } }
 	);
+
+	const description = translate( 'Get a free domain with an annual plan.' );
 	// This banner upsells the ability to install free and paid plugins on a Business plan.
 	return (
 		<UpsellNudge
+			description={ description }
```

Comment:
Possibly want compactButton = false here, the upgrade cta looks a little small as is.

```suggestion
			description={ description }
			compactButton={ false }
```

<img width=350 src=https://github.com/user-attachments/assets/48a8fa22-71fc-4a2c-b88c-cb74b8a39f0f />
<img width=350 src=https://github.com/user-attachments/assets/fe8a36f6-ccd5-489c-bfab-f72470abb99a />

Reply:
Thanks for checking Louis. Addressed in [7ff3195](https://github.com/Automattic/wp-calypso/pull/101049/commits/7ff3195adde53d5f6fb26c55249da63862123848)